stdcommands/undelete/undelete.go

```diff
@@ -13,14 +13,15 @@ import (
 var Command = &commands.YAGCommand{
 	CmdCategory:     commands.CategoryTool,
 	Name:            "Undelete",
-	Aliases:         []string{"ud"},
-	Description:     "Views the first 10 recent deleted messages. By default, only the current user's deleted messages will show.",
+	Aliases:         []string{"ud", "snipe"},
+	Description:     "Views the recent deleted messages. By default, only the current user's deleted messages will show.",
 	LongDescription: "You can use the `-a` flag to view all users delete messages, or `-u` to view a specified user's deleted messages.\nBoth `-a` and `-u` require Manage Messages permission.\nNote: `-u` overrides `-a` meaning even though `-a` might've been specified along with `-u` only messages from the user provided using `-u` will be shown.",
 	RequiredArgs:    0,
 	ArgSwitches: []*dcmd.ArgDef{
 		{Name: "a", Help: "from all users"},
 		{Name: "u", Help: "from a specific user", Type: dcmd.UserID, Default: 0},
 		{Name: "channel", Help: "Optional target channel", Type: dcmd.Channel},
+		{Name: "count", Help: "Number of messages to show, Min: 0, Max 10", Type: dcmd.Int, Default: 10},
```

Comment:
Why min 0? If going to set to 1 anyways why allow them to input an invalid value?